docs/src/content/docs/core/define-integration.mdx

```diff
@@ -22,13 +21,12 @@ export default defineIntegration({
         /** A comment */
         foo: z.string().optional().default("bar"),
     }),
-    plugins: [...corePlugins],
     setup({ options }) {
         const { resolve } = createResolver(import.meta.url);
 
         return {
-            "astro:config:setup": ({ watchIntegration }) => {
-                watchIntegration(resolve())
+            "astro:config:setup": (params) => {
+                watchIntegration(params, resolve())
```

Comment:
Here is the plan regarding it:
- Find a way to get HMR working in the template repo
- Export what's needed from aik (vite plugin, integration whatever)
- Rename watchIntegration to watchDirectory, I think it still has usecases
- Remove watchIntegration

wdyt? So I think if it's not in this this PR it could still be in 0.8 if Luiz finds a way or otherwise that will be in 0.9

Reply:
Yeah I mean that sounds like a great plan! I'm definitely on board with that!

But I did just mean to remove _this_ specific example and replace with something else!

So change the first example in the docs from `watchIntegration` which we'll be removing soon enough. Maybe to change it to `addDts` or something?

Reply:
aah gotcha yeah sounds good!